docs/ProtectedBuildGuide.md

```diff
@@ -139,3 +140,14 @@ The following sections give detailed explanation regarding some of the steps / c
 - The user heap starts at the end of the user bss section and spans the entire remaining space in the user ram region.
 - However, if the chipset has a memory configuration that is different from the default one or if the vendor would like to initialize the kernel and user heaps at different locations or sizes, then the vendor can implement new versions of the APIs in a chipset specific file such as CHIPSET_allocateheap.c and build this new file instead of up_allocate.c file.
 - The vendor needs to take care that the prototypes of the APIs are not changed in their new implementation.
+
+## Available APIs
+Some APIs like scheduler APIs can't be used on user application binary with Protected build because kernel and user binaries are separated.  
```

Comment:
Some scheduler APIs like sched_getprioritymax, task_setcanceltype can be used from user... So I think we'd better change the example...

Reply:
I've mentioned exact one example API (sched_gettcb API) for that.